src/components/Check/Check.scss

```diff
@@ -49,19 +61,34 @@ $checkBoxHeight: 20px;
 }
 
 .ms-Check-background {
+  font-family: 'FabricMDL2Icons';
+  position: absolute;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+
   &:before {
+    content: '\E91F';
+    color: #ffffff;
+  }
+
+  &:after {
+    content: '\EA3A';
+    color: $ms-color-neutralTertiaryAlt;
+  }
+
+  &:before,
+  &:after {
+    @include left(-2px);
```

Comment:
And also, in that screenshot, it would seem that if you nudge the checkmark over, it will be misaligned with the select all checkmark...